BGAnimations/ScreenSelectMusic overlay/PaneDisplay.lua

```diff
@@ -106,13 +115,21 @@ local GetScoresRequestProcessor = function(res, master)
 					end
 
 					if gsEntry["isSelf"] then
-						SetNameAndScore(
-							GetMachineTag(gsEntry),
-							string.format("%.2f%%", gsEntry["score"]/100),
-							playerName,
-							playerScore
-						)
-						personalRecordSet = true
+						-- Let's check if the GS high score is higher than the local high score
+						local player = PlayerNumber[i]
+						local localScore, localName = GetLocalScoreAndNameForPlayer(player)
+						local gsScore = gsEntry["score"]
+						localScore = localScore and tonumber(localScore:gsub("%%", "") * 100) or nil
```

Comment:
I do think it's a little weird that we're getting the score, formatting it correctly, and then unformatting it to get the value again, and then reformatting it below. But you don't have to fix that. I'll revisit it later :) Changes look good to me!